eng/performance/benchmark_jobs.yml

```diff
@@ -22,6 +23,12 @@ jobs:
         displayName: '${{ parameters.osName }} ${{ parameters.osVersion }} ${{ parameters.architecture }} ${{ parameters.kind }}'
         timeoutInMinutes: 320
         variables:
+        - ${{ if eq(parameters.osName, 'windows') }}:
+          - name: SharedHelixPreCommand
+            value: 'set PERFLAB_INLAB=1;set PERFLAB_HASH=$(Build.SourceVersion);set PERFLAB_REPO=$(Build.Repository.Name);set PERFLAB_BRANCH=$(Build.SourceBranch);set PERFLAB_PERFHASH=$(Build.SourceVersion);set PERFLAB_QUEUE=${{ parameters.queue }};set PERFLAB_BUILDNUM=$(Build.BuildNumber);set PERFLAB_BUILDARCH=${{parameters.architecture}};set PERFLAB_LOCALE=${{parameters.locale}}'
+        - ${{ if ne(parameters.osName, 'windows') }}:
+          - name: SharedHelixPreCommand
+            value: 'export PERFLAB_INLAB=1;export PERFLAB_HASH=$(Build.SourceVersion);export PERFLAB_REPO=$(Build.Repository.Name);export PERFLAB_BRANCH=$(Build.SourceBranch);export PERFLAB_PERFHASH=$(Build.SourceVersion);export PERFLAB_QUEUE=${{ parameters.queue }};export PERFLAB_BUILDNUM=$(Build.BuildNumber);export PERFLAB_BUILDARCH=${{parameters.architecture}};export PERFLAB_LOCALE=${{parameters.locale}}'
```

Comment:
PERFLAB_HASH is the hash of the product, which I don't think we know right now on the helix build machine. This is a thing we can pull in later hopefully. PERFLAB_REPO - I think that's going to say "performance" in this example but it should say "core-sdk"?

Reply:
Hm. We were going to use the repo for finding the results containers, which won’t work if this is core-sdk... tricky

Reply:
I changed it, for now, to be dotnet/core-sdk. We'll want to think about this further though.